README.md

```diff
@@ -27,59 +27,63 @@
+
+One of the goals of this library is to make it easy to use. We aim for naming, documentation and usability of the API to be first class.
 
 ### **Flexible Content Loading System**
-**Velaptor** has the ability to load texture atlas data for its rendering as well as single image and sound content.  The content loading system is flexible and has an API with the ability to be extended to load custom content for your media applications and games.
+
+**Velaptor** has the ability to load texture atlas data for its rendering as well as single image and sound content. The content loading system is flexible and has an API with the ability to be extended to load custom content for your media applications and games.
 
 ### **Content Caching**
-Loaded content such as images and sounds are cached for better performance.  If the same content is attempting to be loaded from its source after it has already been loaded, it will be used from memory instead of being reloaded. You can also load content by building your own content loader. 
+
+Loaded content such as images and sounds are cached for better performance. If the same content is attempting to be loaded from its source after it has already been loaded, it will be used from memory instead of being reloaded. You can also load content by building your own content loader.
 
 <h2 style="font-weight:bold;" align="center">📽️ Feature Demo 📽️</h2>
 
 <div align="center">
 
 [![FeatureDemoVideo](https://raw.githubusercontent.com/KinsonDigital/Velaptor/preview/Images/demo-img.jpg)](https://www.youtube.com/watch?v=rcKi-eWeUuo)
+
 </div>
 
 <br/>
 
 <h2 style="font-weight:bold;" align="center">📃 Documentation and Resources 📃</h2>
 
-  Full API documentation and tutorials can be found at [docs.velaptor.io](https://docs.velaptor.io). Velaptor is powered by [CASL](https://github.com/KinsonDigital/CASL) and [Silk.NET](https://github.com/dotnet/Silk.NET).
+Full API documentation and tutorials can be found at [docs.velaptor.io](https://docs.velaptor.io). Velaptor is powered by [CASL](https://github.com/KinsonDigital/CASL) and [Silk.NET](https://github.com/dotnet/Silk.NET).
 
 <br/>
 
 <h2 style="font-weight:bold;" align="center">🙏🏼 Contributing 🙏🏼</h2>
 
 Interested in contributing? If so, click [here](https://github.com/KinsonDigital/.github/blob/main/docs/CONTRIBUTING.md) to learn how to contribute your time or [here](https://github.com/sponsors/KinsonDigital) if you are interested in contributing your funds via one-time or recurring donation.
 
-
 <h2 style="font-weight:bold;" align="center">🔧 Maintainers 🔧</h2>
 
 ![x-logo-dark-mode](https://raw.githubusercontent.com/KinsonDigital/.github/main/Images/x-logo-16x16-dark-mode.svg#gh-dark-mode-only)
 ![x-logo-light-mode](https://raw.githubusercontent.com/KinsonDigital/.github/main/Images/x-logo-16x16-light-mode.svg#gh-light-mode-only)
 [Calvin Wilkinson](https://twitter.com/KDCoder) (KinsonDigital GitHub Organization - Owner)
 
-
```

Comment:
Please add a single empty line to this spot.

**Reason:**
I have some special processing going on with the readme file in my CICD system.
I don't think it will break anything to delete this line, but I would rather not take the chance.